week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,12 +1,30 @@
 // Implement the missing functionality according to the task description.
 // Hint: Remember which function you need to call when it's time to alarm.
 
-function setAlarm() { }
+function setAlarm() {
+  let remainingTime = document.getElementById("timeRemaining");
+  let input = document.getElementById("alarmSet").value;
+
+  let interval = setInterval(function () {
+    if (input > 0 && input < 10) {
+      remainingTime.innerText = `Time Remaining: 00:0${input}`;
+      input--;
+    } else if (input >= 10) {
+      remainingTime.innerText = `Time Remaining: 00:${input}`;
+      input--;
+    } else if (input === 0) {
+      remainingTime.innerText = `Time Remaining: 00:0${input}`;
+      clearInterval(interval);
+      playAlarm();
+    }
+  }, 1000);
+}
 
```

Comment:
The alarm is not working. When set button and stop button are clicked,  they are not setting or stopping the alarm. FIxe the bug there